sql/src/test/java/org/apache/druid/sql/calcite/CalciteWindowQueryTest.java

```diff
@@ -278,6 +278,52 @@ public void testWindow()
         .run();
   }
 
+  @Test
+  public void testWindowAllBoundsCombination()
+  {
+    testBuilder()
+        .sql("select\n"
+             + "cityName,\n"
+             + "count(*) over (partition by cityName order by countryName rows between unbounded preceding and 1 preceding) c1,\n"
```

Comment:
I think it would be usefull to have a set of benchmarks to excercise these things - and see how this changes things